lib/cli/commands/serve.js

```diff
@@ -46,8 +46,12 @@ serve.builder = function(cli) {
 			"Listen to port 1337 and launch default browser with http://localhost:1337/test/QUnit.html");
 };
 
-serve.handler = function(argv) {
+serve.openUrl = async function(browserUrl) {
 	const opn = require("opn");
+	return opn(browserUrl);
+};
+
+serve.handler = function(argv = {}) {
```

Comment:
Why the defaulting to empty object? argv should always be supplied shouldn't it?